apps/ledger-live-desktop/src/main/window-lifecycle.ts

```diff
@@ -112,6 +118,7 @@ export async function createMainWindow({ dimensions, positions }: any, settings:
   mainWindow.on("closed", () => {
     mainWindow = null;
   });
+  // @ts-expect-error Removed in Electron 22. See: https://www.electronjs.org/docs/latest/breaking-changes#removed-webcontents-new-window-event
```

Comment:
so it's possibly dead code?

Reply:
worse, it's a piece of code that was working with electron 22- and that has not been migrated over to electron 22+ 

I am focusing on typing so I did not touch it, but I will add a `// TODO:` comment to outline it. (I thought we were using electron 21 and did not double check 🤦)

https://www.electronjs.org/docs/latest/breaking-changes#removed-webcontents-new-window-event